### PR TITLE
Support CoreDNS 1.8

### PR DESCRIPTION
**Pull request by jplot**
Closes https://github.com/networkservicemesh/fanout/issues/30

### CODE REVIEW

File: utils.go [hunk 1]
```diff
@@ -35,40 +35,42 @@ func logErrIfNotNil(err error) {
...
+	port, _ := strconv.ParseUint(p, 10, 32) // same here
+	ip := net.ParseIP(h)
 
-	if protocol == "tcp" {
-		b.SocketProto = tap.SocketProtocol_TCP
-	} else {
-		b.SocketProto = tap.SocketProtocol_UDP
-	}
+	var ta net.Addr = &net.UDPAddr{IP: ip, Port: int(port)}
+	t := f.net // state.Proto() ?
```
Comment (jplot) [inline utils.go:47]: I have a doubt here, would it not be better to use `state.Proto()`?

File: utils.go [hunk 2]
```diff
@@ -35,40 +35,42 @@ func logErrIfNotNil(err error) {
 	log.Error(err)
 }
 
-func toDnstap(ctx context.Context, host, protocol string, state *request.Request, reply *dns.Msg, start time.Time) error {
-	tapper := dnstap.TapperFromContext(ctx)
-	if tapper == nil {
-		return nil
-	}
+func toDnstap(f *Fanout, host string, state *request.Request, reply *dns.Msg, start time.Time) {
```
Comment (denis-tingajkin) [inline utils.go:38]: This looks like sync up with https://github.com/coredns/coredns/blob/master/plugin/forward/dnstap.go#L16-L58

But it should be fine for now.

@miekg Let us know if we can make some stuff from the forward plugin reusable.

**Review by denis-tingajkin — APPROVED** [review]
@jplot Please pass linter and re-commit your changes with flag `-s` to pass DCO check.

**Comment by denis-tingajkin** [comment]
Note: [Linter job](https://github.com/networkservicemesh/fanout/pull/31/checks?check_run_id=1556447278) has failed with: 
```
utils.go:53:21: Error return value of `msg.SetQueryAddress` is not checked (errcheck)
	msg.SetQueryAddress(q, ta)
	                   ^
utils.go:71:22: Error return value of `msg.SetQueryAddress` is not checked (errcheck)
```
Also you can run linter locally via `golangci-lint`

**Review by denis-tingajkin — APPROVED** [review]

**Comment by denis-tingajkin** [comment]
@jplot Thanks!

**Pull request merged**